ios/use_react_native-0.70.rb

```diff
@@ -12,6 +12,12 @@ def include_react_native!(options)
 
   require_relative(File.join(project_root, react_native, 'scripts', 'react_native_pods'))
 
+  # TODO: Remove this block when react-native-macos catches up to core.
```

Comment:
Catches up to 0.70?

Reply:
Eh, this already merged, so whatever :P

Reply:
Yes, that's what I intended to say 😄  Not catch up in general.